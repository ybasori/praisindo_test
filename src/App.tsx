import React from "react";
import Home from "./Pages/Home/Home";
import "@fortawesome/fontawesome-free/css/all.min.css";
import { Provider } from "react-redux";
import { store } from "./Redux/store";

function App() {
  return (
    <Provider store={store}>
      <Home />
    </Provider>
  );
}

export default App;
