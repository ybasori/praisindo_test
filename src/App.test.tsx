import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock("axios", () => ({
  data: { response: { docs: [] } },
}));

test("renders learn react link", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
