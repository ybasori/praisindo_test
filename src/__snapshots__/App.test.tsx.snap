// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="container"
    >
      <form
        class="container"
      >
        <input
          autocomplete="off"
          class="input"
          name="keyword"
          value=""
        />
        <button
          class="btn"
        >
          <i
            class="fas fa-search"
          />
        </button>
      </form>
      <div
        class="artilce-list"
      >
        <div
          class="container"
        >
          <div
            class="loading"
          >
            <i
              class="fas fa-circle-notch fa-spin"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="to-top-btn"
    >
      <i
        class="fas fa-chevron-up"
      />
    </div>
  </div>
</div>
`;
